content/sklearn/concepts/gaussian-processes/gaussian-processes.md

```diff
@@ -0,0 +1,56 @@
+---
+Title: 'Gaussian Processes'
+Description: 'Gaussian Processes are a supervised learning algorithm that predicts outcomes as distributions, assuming that any set of input points follows a joint Gaussian distribution.'
```

Comment:
``` suggestion
Description: 'Gaussian Processes is a supervised learning framework. that predicts outcomes as distributions, assuming that any set of input points follows a joint Gaussian distribution.'
```
The use of the word `framework` would be more efficient than using the word `algorithm` as the term algorithm suggests that the Gaussian Processes are a single algorithm which is not accurate they are a framework that can be used in various supervised learning models.